scripts/portal-upgrade.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e # exit on first error
+
+# put the server down for maintenance
+. ${cwd}/portal-down.sh
```

Comment:
If I hadn't read the code I'd be more than likely to either call this script from the home dir or just alias something to its full path. Using `${cwd}` is not safe.

Reply:
you are right! I totally forgot to add this 👇  when refactoring, adding it back

```
# get current working directory (pwd doesn't cut it)
cwd=$(cd -P -- "$(dirname -- "$0")" && pwd -P)```

Reply:
I have no clue what this does but I'll trust that you know what you're doing. 👍